### PR TITLE
Add guidelines to enable monitoring for a Kyma component

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:
- Added guidelines  to enable monitoring for a Kyma component

**Related issue(s)**
Fixes this [issue](https://github.com/kyma-project/kyma/issues/1916)

### CODE REVIEW

File: guidelines/add-monitoring-to-component-docs.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Add monitoring to a Kyma component
+
+## Enable dashboard in Grafana
+- Add label `kyma-grafana: enabled` along with label `app: <value>` to the **PodTemplate**. Make sure label `app: <value>` is added to either `Deployment` or `Statefulset` spec as well. Then a pre-packaged dashboard with [RED](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture) and [USE](http://www.brendangregg.com/usemethod.html) metrics for the application gets enabled. The dashboard is defined [here](link).
```
Comment (bszwarc) [inline guidelines/add-monitoring-to-component-docs.md]:
```suggestion
- Add the `kyma-grafana: enabled` and the  `app: <value>` label to the **PodTemplate**.  Make sure you add the `app: <value>` label either to `Deployment` or `Statefulset` specification as well. Performing this step enables the pre-packaged dashboard with [RED](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture) and [USE](http://www.brendangregg.com/usemethod.html) metrics for the application. The dashboard is defined [here](link).
```

File: guidelines/add-monitoring-to-component-docs.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Add label `kyma
-grafana: enabled` along with label `app: <value>` to the **PodTemplate**. Make sure label `app: <value>` is added to either `Deployment` or `Statefulset` spec as well. Then a pre-packaged dashboard with [RED](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture) and [USE](http://www.brendangregg.com/usemethod.html) metrics for the application gets enabled. The dashboard is defined [here](link).
+- The dashboard is available as `Services
` where the application is visible with the value of it's **app** label in its concerned namespace.
+
+
+## Enable alerts
+- Add label `kyma
-alerts: enabled` along with label `app: <value>` to your
 Kyma component. Make sure label `app: <value>` is added to either `Deployment` or `Statefulset` spec as well. Then a pre-packaged sets of alert rules get enabled. The alert rules are defined [here](link).
```
Comment (bszwarc) [inline guidelines/add-monitoring-to-component-docs.md]:
```suggestion
- Add the `kyma-alerts: enabled` and the `app: <value>` label to your Kyma component. Make sure you add the `app: <value>` label either to `Deployment` or `Statefulset` specification as well. Performing this step enables pre-packaged sets of alert rules. The alert rules are defined [here](link).
```

File: guidelines/add-monitoring-to-component-docs.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## Sample application where monitoring is enabled
```
Comment (bszwarc) [inline guidelines/add-monitoring-to-component-docs.md]:
```suggestion
## Sample application with enables monitoring
```

File: guidelines/add-monitoring-to-component-docs.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+spec:
+  ports:
+  - name: http
+    port: 5000
+    protocol: TCP
+    targetPort: 5000
+  selector:
+    app: demo-service
+```
+- Dashboard for this application will be available in Grafana `Services` dashboard as `demo-service` in namespace `stage`.
```
Comment (bszwarc) [inline guidelines/add-monitoring-to-component-docs.md]:
```suggestion
- The dashboard for this application is available in Grafana under **General > Services**.  To display it, select `demo-service` in the `stage` Namespace.
```

File: guidelines/add-monitoring-to-component-docs.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+# Add monitoring to a Kyma component
+
+## Enable dashboard in Grafana
+- Add label `kyma-grafana: enabled` along with label `app: <value>` to the **PodTemplate**. Make sure label `app: <value>` is added to either `Deployment` or `Statefulset` spec as well. Then a pre-packaged dashboard with [RED](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture) and [USE](http://www.brendangregg.com/usemethod.html) metrics for the application gets enabled. The dashboard is defined [here](link).
+- The dashboard is available as `Services` where the application is visible with the value of it's **app** label in its concerned namespace.
```
Comment (bszwarc) [inline guidelines/add-monitoring-to-component-docs.md]:
```suggestion
- To see the dashboard, go to  **General > Services** in Grafana. You find the application using the value of its `app` label and the Namespace you specified in in your configuration.
```

File: guidelines/add-monitoring-to-component-docs.md (hunk 1 above)
Comment (sayanh) [reply guidelines/add-monitoring-to-component-docs.md]: done

File: guidelines/add-monitoring-to-component-docs.md (hunk 3 above)
Comment (sayanh) [reply guidelines/add-monitoring-to-component-docs.md]: done

File: guidelines/add-monitoring-to-component-docs.md (hunk 4 above)
Comment (sayanh) [reply guidelines/add-monitoring-to-component-docs.md]: done

File: guidelines/add-monitoring-to-component-docs.md (hunk 5 above)
Comment (sayanh) [reply guidelines/add-monitoring-to-component-docs.md]: done

File: guidelines/add-monitoring-to-component-docs.md (hunk 2 above)
Comment (sayanh) [reply guidelines/add-monitoring-to-component-docs.md]: done

File: guidelines/add-monitoring-to-component-docs.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
+# Add monitoring to a Kyma component
+
+## Enable dashboard in Grafana
+- Add the `kyma-grafana: enabled` and the `app: <value>` label to the **PodTemplate**. Make sure you add the `app: <value>` label either to `Deployment` or `Statefulset` specification as well. Performing this step enables the pre-packaged dashboard with [RED](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture) and [USE](http://www.brendangregg.com/usemethod.html) metrics for the application. The dashboard is defined [here](link).
+- To see the dashboard, go to **General > Services** in Grafana. You find the application using the value of its `app` label and the Namespace you specified in your configuration.
```
Comment (bszwarc) [inline guidelines/add-monitoring-to-component-docs.md]:
```suggestion
- To see the dashboard, go to **General > Services** in Grafana. There, you can find your application using the values for the `app` label and the Namespace you specified in your configuration.
```

**Pull request merged**